tests/tests.nix

```diff
@@ -44,33 +44,19 @@ in
       inherit root;
       include = [
         "package.json"
-        "src"
-        (nix-filter.matchExt "js")
-      ];
-    };
-    expected = [
-      "package.json"
-      "src"
-      "src/main.js"
-    ];
-  };
-
-  with-matchExt2 = rec {
-    root = ./fixture1;
-    actual = nix-filter {
-      inherit root;
-      include = [
-        "package.json"
```

Comment:
With the current behaviour, I thought the two tests were too similar so I tried to merge them.